omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,103 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated read-only group to host the raw data underlying the
+   publication (see :doc:`cli/usergroup`)
+#. Add all the authors of the paper to this new group.
+#. If you have not already done so, configure the :ref:`public_user`.
+#. Add the public user as a member of the newly created read-only group to
+   make all the data publicly available.
+
+Data setup
+----------
+
+#. Move the original images into the dedicated group using OMERO.web or
+   :doc:`OMERO.cli </users/cli/chgrp>`. The CLI is best used where Images or
+   Datasets are cross-linked to other Datasets or Projects in the original
+   group. The command ``bin/omero chgrp Project:$ID --include Dataset,Image``
+   cuts the cross-links in the original group and preserves the
+   Project/Dataset/Image hierarchy prepared for the move by the author.
+#. If you have used OMERO.figure to create your figures for publication, you
+   can always find the original data by using the 'info' tab, as shown in the
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
+#. Having all the data belong to one user simplifies the UI experience for
+   public users. If necessary, ownership of data can be transferred using the
+   'Chown' privilege (see :doc:`restricted-admins` and
+   :doc:`/users/cli/chown`).
+
+Configuring OMERO.web
+---------------------
+
+#. Configure the OMERO.web :ref:`filter on the public user <public.url_filter>` by setting
+   :property:`omero.web.public.url_filter` to allow 'webclient' so that the
+   full webclient is visible for the public user, and thus the Data tree with
+   Projects and Datasets is also browsable, as well as the Tags tab and the
+   full image viewer.
+#. In order to have an automatically logged-in public user while still giving
+   your existing OMERO users an unchanged user experience (i.e. not
+   automatically logging them in as the public user), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at your existing
+   OMERO.server.
```

Comment:
If users have been following these instructions in order (instead of reading everything first), they may start to configure the public user on their existing OMERO.web before reading that the suggestion that they should create another OMERO.web server. From above: "If you have not already done so, configure the `public_user`". Even if they have read everything through first, we need to make it clearer that the public user is configured on the new OMERO.web and not the existing one.

Reply:
:+1: This is likely the time for us to start adding simple drawings of the various scenarios.

Basic OMERO:
```
 p <--> o <--> w
```
This setup:

```
 p <--> o <--> w
        |
        V
      pub. w
```

Reply:
I'm not following that diagram @joshmoore what's 'p'? (assuming o=OMERO.server and w=OMERO.web)

Reply:
maybe postgres?

Reply:
@will-moore - they may not always want to configure a new OMERO.server - a la the webgateway-only "original nightshade" setup. In that case, it will be "configure public user on your existing server.

Reply:
@kennethgillen Agreed, for most admins it'll be a major effort to setup and maintain another OMERO.web. We probably need more details about how disruptive it is to have public webclient for other non-public users. Ideally we could address most of the issues (e.g. https://github.com/openmicroscopy/openmicroscopy/pull/5557 and https://github.com/openmicroscopy/openmicroscopy/pull/5558) and not need another OMERO.web.